fairlearn/postprocessing/_roc_curve_utilities.py

```diff
@@ -88,11 +117,25 @@ def _interpolate_curve(data, x_col, y_col, content_col, x_grid):
     return pd.DataFrame(dict_list)[[x_col, y_col, P0_KEY, content_col_0, P1_KEY, content_col_1]]
 
 
-def _calculate_roc_points(data, attribute, flip=True):
+def _calculate_roc_points(data, sensitive_feature_value, flip=True):
+    """ Calculates the ROC points from the scores and labels by iterating through all possible
+    thresholds that could be set based on the available scores.
+
+    :param data: the DataFrame containing scores and labels
+    :type data: pandas.DataFrame
+    :param sensitive_feature_value: the sensitive feature value of the samples provided in `data`
+    :type sensitive_feature_value: str or int
+    :param flip: if True flip points below the ROC diagonal into points above by applying negative
+        weights; if False does not allow flipping; default True
+    :type flip: bool
+    :return: the ROC curve points with their corresponding threshold operations
+    :rtype: pandas.DataFrame
+    """
     scores, labels, n, n_positive, n_negative = _get_scores_labels_and_counts(data)
 
     if n_positive == 0 or n_negative == 0:
-        raise ValueError("Degenerate labels for attribute value {}".format(attribute))
+        raise ValueError("Degenerate labels for sensitive feature value {}" \
```

Comment:
Should this string be a constant up top?

Reply:
added a test case for this case too which was missing so far :O